python/tvm/auto_scheduler/feature.py

```diff
@@ -252,3 +253,78 @@ def get_per_store_feature_names(max_n_bufs: Optional[int] = None) -> List[str]:
         The names of elements in the flatten feature vector
     """
     return _ffi_api.GetPerStoreFeatureNames(max_n_bufs or DEFAULT_MAX_N_BUFS)
+
+
+def primfunc_features(
```

Comment:
Seems like `get_features_from_primfunc` or something in this direction would be more precise.

Reply:
Renamed to `features_from_primfunc`